languages/csharp/exercises/concept/Interfaces/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+In this exercise you will be implement a sofware module for document management system.
```

Comment:
I don't fully understand how this exercise would work. Could you an an example implementation file that contains what code you think the student should be writing?

Reply:
My intention was let students to know how to implement multiple interfaces in a class  while keeping explict and implicit concepts in mind.

For an instance, think about a  document management module wants you to export the content into pdf or word?
So you create 2 contracts/interfaces to represent Pdf and Word types. (See below)
So the document module will consume your implementation base on the export action.
Either Pdf or Word.
Hope the below sample would help you to understand my intention.
Please feel free to add your valuable comments.

`
namespace Test
{
     public static void Main() 
     {
        var document = new Document();
        // cast. alternatively you can use Is or As operators for secure casting.
        var data = ((IPdf)document).Export();
    }

    public class Document : IPdf, IWord
    {
        string IPdf.Export()
        {
            return "Pdf data";
        }

        string IWord.Export()
        {
            return "Word data";
        }
    }

    public interface IPdf 
    {
        string Export();
    }

    public interface IWord
    {
        string Export();
    }
}
`

Reply:
I understand what you're getting at. Regarding the contents of the exercise, here are some suggestions:

- Don't have the exercise require explicit interface implementations. In the majority of the cases, interfaces are not implemented through explicit interfaces implementations. I'd prefer the exercise to just use regular, non-explicit implementations.
- Your story/theme is a real-life example, but I don't see how a small, basic Concept Exercise  would allow for implementing code to output to a PDF or to Word. For the object-oriented exercises, we've used two themes: [cars](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/classes/.docs/instructions.md) and [RPG's games](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/inheritance/.docs/instructions.md). It would be great if this exercise could re-use one of those themes, whichever works best for this exercise.